changelogs/unreleased/4007-ywk253100

```diff
@@ -0,0 +1 @@
+Wait the namespace deletion completed before removing the CRDs when uninstalling the velero. This removes the `--wait` flag of the `uninstall` command
```

Comment:
Grammar nits - 
Wait for the namespace to be deleted before removing the CRDs during uninstall.
would be better for the first sentence